conbench/templates/app.html

```diff
@@ -136,8 +136,36 @@ <h4 class="modal-title"><span class="glyphicon glyphicon-trash submenu"></span>&
             table.search("{{ search_value }}").draw();
         {% endif %}
     });
+
 }
 
+// requires two inputs: table_id & column_indexes
+function specific_column_search(data_table, column_indexes){
```

Comment:
Could you maybe add a code comment specifying the behavior of this function?

Something like "identify the set of values in ...., and then ..."